iphone/iphone/Titanium.plist

```diff
@@ -57,5 +57,23 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+    <key>UIApplicationShortcutItems</key>
```

Comment:
But on devices that do support it, won't it make this shortcut appear without the developer expecting it to?